prod-sample/sample-modules/frank-lloyd-root/0.0.3/package.json

```diff
@@ -20,12 +20,20 @@
     "react": "^16.4.2",
     "react-helmet": "^5.2.1",
     "react-immutable-proptypes": "^2.1.0",
+    "react-intl": "^3.6.0",
```

Comment:
What is the reasoning for adding `react-intl` it isn't used within `frank-lloyd-root`

Reply:
For the integration tests, I'm using `cultured-frankie` v`0.0.1` and does require `react-intl` as a `requiredExternal`, so I've added it here.